config/1.0.0/sample_config_1.0.0.json

```diff
@@ -136,6 +136,19 @@
         }
       ]
     },
+    {
```

Comment:
I've basically migrated all the supplements to overrides. The reason this one shows up in particular is that I moved it to be closer to the other SMAP L3 overrides, so the location in the file also changed.